test/libsolidity/semanticTests/abiencodedecode/abi_encode_call_uint_bytes.sol

```diff
@@ -0,0 +1,20 @@
+contract C {
+	function removeSignature(bytes calldata x) external pure returns (bytes memory) {
+		return x[4:];
+	}
```

Comment:
I can get away with this, since gas is irrelevant, but we should really think about how we can do memory slices :-).